rllib/evaluation/env_runner_v2.py

```diff
@@ -821,6 +820,9 @@ def _handle_done_episode(
             new_episode: EpisodeV2 = self._active_episodes[env_id]
             self._call_on_episode_start(new_episode, env_id)
 
+            # Step after adding initial obs. This will give us 0 env and agent step.
+            new_episode.step()
```

Comment:
also, can you move this slightly below to be after line 853, where we ``add_init_obs()``?
thanks.

Reply:
hmm, maybe you are intentionally trying to step() before add_obs()?
can we chat quickly? wonder if we should fix this another way.

Reply:
i am intentionally doing that yes